subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/listener/AdbDeviceEventsListener.kt

```diff
@@ -0,0 +1,41 @@
+package com.avito.runner.service.worker.device.adb.listener
+
+import java.nio.file.Path
+
+interface AdbDeviceEventsListener {
+    fun getSdkPropertySuccess(attempt: Int, api: Int)
```

Comment:
We should have a device as a first parameter in each function

Reply:
yes, i agree, adding